interpreter/syntax/operators.ml

```diff
@@ -227,6 +227,8 @@ let v128_bitselect = Ternary (V128Op.Bitselect)
 let i8x16_neg = Unary (V128 (V128Op.I8x16 V128Op.Neg))
 let i8x16_any_true = Test (V128 (V128Op.I8x16 V128Op.AnyTrue))
 let i8x16_all_true = Test (V128 (V128Op.I8x16 V128Op.AllTrue))
+let i8x16_shl = Shift (V128Op.I8x16 V128Op.Shl)
+let i8x16_shr_s = Shift (V128Op.I8x16 V128Op.ShrS)
```

Comment:
Nit: could shorten this to `Shift V128Op.(I8x16 Shl)`. Similarly many of the other cases.

Reply:
Good tip, I'll change only this for now, then make the change for the rest (to reduce some local merge conflicts).